src/main/java/org/tron/common/utils/Utils.java

```diff
@@ -618,6 +620,12 @@ public static JSONObject printTransactionToJSON(Transaction transaction, boolean
                             JSONObject.parseObject(
                                     JsonFormat.printToString(unDelegateResourceContract, selfType));
                     break;
+                  case CancelAllUnfreezeV2Contract:
+                    CancelAllUnfreezeV2Contract cancelAllUnfreezeV2Contract =
+                        contractParameter.unpack(CancelAllUnfreezeV2Contract.class);
+                    contractJson = JSON.parseObject(
```

Comment:
Why not use JSONObject.parseObject like everywhere else.

Reply:
ParseObject is a static method of the parent class, and it is recommended to call static methods directly through the parent class.